src/System.Net.Security/tests/FunctionalTests/ServerAsyncAuthenticateTest.cs

```diff
@@ -0,0 +1,219 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Net.Sockets;
+using System.Net.Test.Common;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Security.Tests
+{
+    public class ServerAsyncAuthenticateTest
+    {
+        private readonly ITestOutputHelper _log;
+        private readonly X509Certificate2 _serverCertificate;
+
+        public ServerAsyncAuthenticateTest()
+        {
+            _log = TestLogging.GetInstance();
```

Comment:
The default XUnit ITestOutputHelper doesn't work if called outside the context of a test. 

TestLogging is System.Net specific and uses ETW for fast and reliable logging regardless of context. See https://github.com/dotnet/corefx/blob/master/Documentation/debugging/windows-instructions.md for information on how to collect these traces.